firstvoices/backend/tasks/visibility_tasks.py

```diff
@@ -69,7 +69,7 @@ def bulk_change_visibility(job_instance_id):
     # Resume search indexing for site, + reindex entire site
     indexing_paused_feature[0].is_enabled = False
     indexing_paused_feature[0].save()
-    sync_all_site_content_in_indexes(site)
+    sync_all_site_content_in_indexes(site.id)
```

Comment:
Note: this call to the `sync_all_media_site_content_in_indexes` task is used without apply_async as it is already in the `bulk_change_visibility` async task

Reply:
Not sure, but do we want to use a apply_async call for consistency in calling/logging async tasks ?

Logs from this call will be under the parent task's id, though we will still have the method name in the log for debugging.
Sample log format for reference:
`[timestamp: INFO/PoolWorker-X] backend.tasks.send_email_task[task-id]: Task started.`

Reply:
adding apply_async for log consistency, thanks!